cea/demand/thermal_loads.py

```diff
@@ -321,13 +323,18 @@ def calc_Qhs_sys(bpr, tsd):
     return tsd
 
 
-def calc_Qhs_Qcs(bpr, date, tsd, use_dynamic_infiltration_calculation, region):
+def calc_Qhs_Qcs(bpr, date, tsd, use_dynamic_infiltration_calculation, region, building_name, config, locator):
     # get ventilation flows
     ventilation_air_flows_simple.calc_m_ve_required(bpr, tsd, region)
     ventilation_air_flows_simple.calc_m_ve_leakage_simple(bpr, tsd)
     # get internal comfort properties
-    tsd = control_heating_cooling_systems.calc_simple_temp_control(tsd, bpr, date.dayofweek)
-    # initialize first previous time step
+    # predefined set points for every given hour can be used to calculate the demand profile for a building
+    # a config flag is used for this, it is present in the config.demand section
+    if config.demand.predefined_hourly_setpoints:
```

Comment:
this clause should be `if predefined_hourly_setpoints_present(...)` and called before entering this function...